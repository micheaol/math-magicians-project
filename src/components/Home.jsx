import React from 'react';

const Home = () => (
  <div className="home-wrapper" data-testid="home">
    <h1>Welcome to our page!</h1>
    <div>
      <p>Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores</p>
      <p>Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores</p>
      <p>Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores</p>
    </div>

    <div>
      <p>Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores</p>
      <p>Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores</p>
      <p>Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores</p>
    </div>

    <div>
      <p>Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores</p>
      <p>Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores</p>
      <p>Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores</p>
    </div>
  </div>
);

export default Home;
