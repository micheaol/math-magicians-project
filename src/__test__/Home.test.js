import { render, screen, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

afterEach(() => {
  cleanup();
});
describe('render home component to the screen', () => {
  test('render home component to the screen', () => {
    render(<Home />);
    const homeElement = screen.getByTestId('home');
    expect(homeElement).toBeInTheDocument();
    expect(homeElement).toContainHTML('<h1>Welcome to our page!</h1>');
  });

  test('matches snapshot', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
