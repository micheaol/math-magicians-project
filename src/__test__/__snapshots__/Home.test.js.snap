// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render home component to the screen matches snapshot 1`] = `
<div
  className="home-wrapper"
  data-testid="home"
>
  <h1>
    Welcome to our page!
  </h1>
  <div>
    <p>
      Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
    </p>
    <p>
      Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
    </p>
    <p>
      Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
    </p>
  </div>
  <div>
    <p>
      Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
    </p>
    <p>
      Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
    </p>
    <p>
      Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
    </p>
  </div>
  <div>
    <p>
      Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
    </p>
    <p>
      Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
    </p>
    <p>
      Lo,ipsumdolorsit amet consecteturadipisicing elit Asperiores adipisicingelit Asperiores
    </p>
  </div>
</div>
`;
